src/io/flutter/view/InspectorPanel.java

```diff
@@ -748,8 +751,23 @@ protected void customizeCellRenderer(JTable table, @Nullable Object value, boole
         node.hasCreationLocation() ? SimpleTextAttributes.REGULAR_BOLD_ATTRIBUTES : SimpleTextAttributes.REGULAR_ATTRIBUTES;
       append(node.getName(), attributes);
 
-      // TODO(pq): add property description.
-      //setToolTipText(...);
+      // Set property description in tooltip.
+      // TODO (pq):
+      //  * consider tooltips for values
+      //  * consider rich navigation hovers (w/ styling and navigable docs)
+      final CompletableFuture<String> propertyDoc = node.getPropertyDoc();
+      final String doc = propertyDoc.getNow(null);
+      if (doc != null) {
+        setToolTipText(doc);
+      }
+      else {
+        AsyncUtils.whenCompleteUiThread(propertyDoc, (String tooltip, Throwable th) -> {
```

Comment:
I think you want to call 
setToolTipText(null) 
or setToolTipText("")
right before awaiting the future or you could show stale tooltips for cases where the inspectorService is overloaded with requests

Reply:
Good idea!